src/Symfony/Component/Validator/Constraints/ImageValidator.php

```diff
@@ -234,4 +238,29 @@ public function validate(mixed $value, Constraint $constraint)
             imagedestroy($resource);
         }
     }
+
+    /**
+     * @param mixed $value
+     *
+     * @return bool
+     */
+    private function isSvg($value)
+    {
+        return 'image/svg+xml' === mime_content_type($value);
```

Comment:
We don't use that function anywhere in the codebase, and we have a Mime component.
I guess we should take inspiration from what we do in `FileValidator` here.

Reply:
Done, also we could not detect the File Mime, I return false and treated it as a normal image.